tests/epyccel/test_epyccel_sets.py

```diff
@@ -364,3 +334,50 @@ def copy_from_arg2(a : 'set[float]'):
     python_result = copy_from_arg2(a)
     assert isinstance(python_result, type(pyccel_result))
     assert python_result == pyccel_result
+
+
+@pytest.mark.parametrize( 'language', (
+        pytest.param("fortran",marks = [
+            pytest.mark.skip(reason="pop method not implemented in fortran"),
+            pytest.mark.fortran]),
+        pytest.param("c", marks = pytest.mark.c),
+        pytest.param("python", marks = pytest.mark.python)
+    )
+)
```

Comment:
You may want to create a new module parameter with a similar name to avoid repetition here. E.g `language` and `python_language` or `c_language` `language_without_fortran`, etc

Reply:
Can you please explain more about `creating a new module parameter with a similar name to avoid repetition`? If you have an example, I would be grateful.

Reply:
The example is the current module fixture which can be found at the top of this file

Reply:
Fixed : 5828c07